homeassistant/components/peco/sensor.py

```diff
@@ -23,16 +27,44 @@
 
 from .const import CONF_COUNTY, DOMAIN, LOGGER, SCAN_INTERVAL
 
+
+@dataclass
+class PECOSensorEntityDescriptionMixin:
+    """Mixin for required keys."""
+
+    value_fn: Callable[[OutageResults], int]
+
+
+@dataclass
+class PECOSensorEntityDescription(
+    SensorEntityDescription, PECOSensorEntityDescriptionMixin
+):
+    """Description for PECO sensor."""
+
+
 PARALLEL_UPDATES: Final = 0
-SENSOR_LIST = (
-    SensorEntityDescription(key="customers_out", name="Customers Out"),
-    SensorEntityDescription(
+SENSOR_LIST: tuple[PECOSensorEntityDescription, ...] = (
+    PECOSensorEntityDescription(
+        key="customers_out",
+        name="Customers Out",
+        value_fn=lambda data: int(data.customers_out),
+    ),
+    PECOSensorEntityDescription(
         key="percent_customers_out",
         name="Percent Customers Out",
         native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda data: int(data.percent_customers_out),
```

Comment:
Why does it need to be casted to `int`? What is the original type?

Reply:
See: <https://github.com/home-assistant/core/pull/68707/files/e4e5e5c438617f1dd1640a4443d789b5b9d5921f..512fac784a2de2865e0027876085dfc854612428#r836201351>